.github/workflows/test_ci.yaml

```diff
@@ -39,8 +44,6 @@ jobs:
             ~/.cache/go-build
           key: go-test-${{ hashFiles('**/go.sum') }}
           restore-keys: go-test-
-      - name: install fio to test with fio plugin
-        run: sudo apt-get -y install fio
```

Comment:
I removed FIO as a dependency of the test suite because I can't prove that I need it, the fio plugin is broken-ish (it doesn't work with fio `3.36`). If I decide to add this test case back, it will be with a python plugin that solely reads and writes to its current working directory without requiring FIO as a dependency.